### DISCUSSION TITLE
SuperGrate also migrate installed programs, printers, desktop ?

### DISCUSSION THREAD

**Discussion by Clemens-Ratte-Polle** [belowaverage-org/SuperGrate#108] [Q&A] [answered]
will SuperGrate also migrate all installed programs, printers, desktop so i can work 100% as before?
I dont find clear explanations.
thx

**Comment by krisdb2009** [comment]
It should be able to do everything USMT can do, you might need to tweek some of the config files in the USMT folder to accomplish this.
https://learn.microsoft.com/en-us/windows/deployment/usmt/usmt-overview

**Reply by Clemens-Ratte-Polle** [reply, marked as answer]
the own desktop and all installed programs and printer and many other are NOT migrated:
https://learn.microsoft.com/de-de/windows/deployment/usmt/usmt-what-does-usmt-migrate
